### ISSUE TITLE
integration with openstreetmap?

### ISSUE THREAD

**Issue by npdoty** [openbenches/openbenches.org#232]
I see in the about blog post that you looked at OpenStreetMap data, and found that it had benches but not memorial information. While very few of the 1.4 million benches on OpenStreetMap do, there are at least a couple thousand that specifically record an "inscription" tag. I use this for memorial benches and I'm confident that many other mappers do too.

Taginfo: https://taginfo.openstreetmap.org/tags/amenity=bench#combinations
2000+ have inscription (and indeed, it's a pretty common co-occurrence for inscriptions, of which over 2% are specifically benches), and 4000+ have description (text descriptions of the bench, which could often include details about the bench including memorial data, although certainly not all will be that).

Would there be an interest in using the OpenStreetMap database as a part of your data model? Or making it easy (or semi-automated) to add your memorial inscriptions as connections to existing OpenStreetMap bench objects? Or even help the user select from existing OSM-tagged benches when they're uploading photos of a memorial bench? Or show OSM-tagged inscription benches on your map when people are looking for benches in their area?

Just seemed like there were opportunities and so I wanted to raise the possible connection, though I'm sure you've thought of some of this before. I'll keep tagging memorial benches on OpenStreetMap and I hope they'll get used by projects like yours.

**Comment by edent** [comment]
Heya, very happy if someone wants to add the OpenBenches data into OSM. We did look into it a few years ago, but there wasn't much interest.  We have an API and a database dump, if that makes it easier. Let me know what format works best.

As for bringing in data to us - we've imported data from Wikipedia, Geograph, and Flickr.   But they all have photos, I don't think OSM does.

**Comment by npdoty** [comment]
Right, OSM data would likely have the text of the inscription, in addition to the geodata, but won't have pictures. Is OpenBenches only tracking memorial benches that have photos of them?

I've asked a bit about doing an import from OpenBenches into OSM and I'll update here with what I learn.

**Comment by edent** [comment]
Heya. Yes, we're just collecting photos at the moment. We don't want to recreate OSM and be a definitive list of every bench.

**Comment by edent** [comment]
A few years later and... this might be happening! https://en.osm.town/@pietervdvn/110078001456634211

**Comment by edent** [comment]
There's now an "Add to OSM" button.
